ui/app/pages/swap/swap-content/swap.utils.js

```diff
@@ -1,51 +1,38 @@
 import {
```

Comment:
Do you need this file? I think alot of its functionality is duplicated in the swapsController